package.json

```diff
@@ -13,6 +13,10 @@
     {
       "name": "Ricardo Bin",
       "email": "ricardohbin@gmail.com"
+    },
+    {
+      "name": "Marc Löhe",
+      "email": "marcloehe@gmail.com"
```

Comment:
Please, can you add yourself in Release History in README (that would be `1.2.1`) instead of here? Because we use this field to main contribuitors only